modules/src/test/java/eu/itesla_project/modules/contigencies/TapChangeActionTest.java

```diff
@@ -40,14 +48,17 @@ public void testContructors() {
     }
 
     @Test
-    public void testToTask() {
+    public void testToTask() throws Exception {
         Network network = PhaseShifterTestCaseFactory.create();
         PhaseTapChanger tapChanger = network.getTwoWindingsTransformer("PS1").getPhaseTapChanger();
         assertEquals(1, tapChanger.getTapPosition());
 
         TapChangeAction action = new TapChangeAction("PS1", 2);
         ModificationTask task = action.toTask();
-        task.modify(network);
+        FileSystem fileSystem = Jimfs.newFileSystem(Configuration.unix());
```

Comment:
fileSystem has to closed. Create the fs in the setUp method of the unit-test and close it in the tearDown method. ComputationManager can also be created in the setUp method